python/tvm/topi/arm_cpu/conv2d_alter_op.py

```diff
@@ -101,9 +102,6 @@ def _alter_conv2d_layout(attrs, inputs, tinfos, out_type):
         # we then assume it's not necessary to alter this op.
         return None
     cfg = dispatch_ctx.query(target, workload)
-    if cfg.is_fallback:  # if is fallback, clear query cache and return None
-        autotvm.task.clear_fallback_cache(target, workload)
-        return None
```

Comment:
I hope it's safe to remove this. The x86 counterpart doesn't have thing like this.

Reply:
How are you testing it ?

Reply:
If I can pass the CI with this change, I assume it is safe.

Reply:
@Mousius  - could you please look at this given you've recently been turning on topi tests on aarch64 ?

Reply:
I need to remove this code because it always makes `alter_layout` nop in the fallback mode. In contrast, in the x86 schedule, `alter_layout` always fires.

Reply:
The CI has passed. I found that Giuseppe's im2col based conv2d implementation can fail to tensorize in the fallback mode, so I partially restored the fallback return path above.